src/utils/jump_utils.jl

```diff
@@ -29,50 +29,51 @@ function jump_value(input::Float64)::Float64
     return input
 end
 
-function to_matrix(array::DenseAxisArray)
+function to_matrix(array::DenseAxisArray{T, 1, K}) where {T, K <: NTuple{1, Any}}
```

Comment:
Similar to to_matrix that accepts a two-dimensional DenseAxisArray, should there be one method specific to Real and a second one that is generic?